Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -30,12 +32,40 @@ - (instancetype)initWithTox:(OCTTox *)tox
     _toxAV = [[OCTToxAV alloc] initWithTox:tox error:nil];
     _toxAV.delegate = self;
 
+    _audioEngine = [OCTAudioEngine new];
+    _audioEngine.toxav = self.toxAV;
+
+    _chatConverter = [OCTConverterChat new];
+
+    _mutableCalls = [NSMutableSet new];
+
     return self;
 }
 
+- (NSSet *)calls
+{
+    return [self.mutableCalls copy];
+}
+
 - (OCTCall *)callToChat:(OCTChat *)chat enableAudio:(BOOL)enableAudio enableVideo:(BOOL)enableVideo
 {
-    return nil;
+    OCTToxAVAudioBitRate audioBitRate = (enableAudio) ? 24000 : kOCTToxAVAudioBitRateDisable;
+    OCTToxAVVideoBitRate videoBitRate = (enableVideo) ? 400 : kOCTToxAVVideoBitRateDisable;
```

Comment:
Please don't use magic numbers. You can create `kDefaultAudioBitRate` or something like constant